lib/run.js

```diff
@@ -245,8 +245,8 @@ function run (opts, cb) {
     }
 
     if (typeof opts.duration === 'string') {
-      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration)
-      else opts.duration = Number(opts.duration.trim())
+      if (/[a-zA-Z]/.exec(opts.duration)) opts.duration = timestring(opts.duration) * 1000
```

Comment:
Would you please change this `timestring(opts.duration) * 1000` to `timestring(opts.duration, 'ms')` if you can, see: https://www.npmjs.com/package/timestring#return-time-value